css/public/style.css

```diff
@@ -438,3 +447,42 @@ detailsitem .select2-container {
 .select2-results .select2-result-label span {
 	cursor: pointer !important;
 }
+
+@media only screen and (max-width: 768px) {
+
+/* full width for message list on mobile */
+.app-content-list {
+	width: 100%;
+	background: white;
+	position: relative;
+	z-index: 100;
+}
+
+/* overlay message detail on top of message list */
+.app-content-detail {
+	background: #fff;
+	width: 100%;
+	left: 0;
+	height: 100%;
+	top: 0;
+	box-shadow: 0 0 100px rgba(100, 100, 100, .9);
+	position: absolute;
+}
+
+button.details-back {
+	display: block !important;
+}
+
+/* end of media query */
+}
+
+.contact-details-wrapper {
```

Comment:
We need `height: 100%;` to avoid having the list poping up under the contactDetail: :)
![contacts owncloud](https://cloud.githubusercontent.com/assets/14975046/14103395/39ab7000-f5a0-11e5-884f-16b80c49e647.png)